### ISSUE TITLE
puzzle hunt

### ISSUE THREAD

**Issue by tivvit** [gdg-garage/garage-trip-5.0#16]
Having a day-long outdoor puzzle hunt at GT is a very long tradition, but all the orgs already exhausted most of the ideas.

Please consult https://www.garage-trip.cz/history and bring your own ideas to life.

Discussion: https://discord.com/channels/750810991897608293/1196202682805928049

**Comment by tivvit** [comment]
@kethzigilbert you mentioned you have some experience right? Would you be interested in preparing this?

**Comment by kethzigilbert** [comment]
I have done small scale scavenger hunts for house parties thats all :D It would be nice if i can team up with someone if possible for this

**Comment by tivvit** [comment]
Definitely, let's wait to see if someone will volunteer, and if not, Eva and I can help. 
Please take a look at the previous ones and we can discuss your ideas and what is necessary for the realization.
